cmd/acr/purge.go

```diff
@@ -48,15 +50,18 @@ const (
   - Delete all tags older than 1 day in the example.azurecr.io registry inside the hello-world repository, with 4 purge tasks running concurrently
 	acr purge -r example --filter "hello-world:.*" --ago 1d --concurrency 4
 	`
-	maxPoolSize              = 32 // The max number of parallel delete requests recommended by ACR server
-	manifestListContentType  = "application/vnd.docker.distribution.manifest.list.v2+json"
-	manifestIndexContentType = "application/vnd.oci.image.index.v1+json"
-	linkHeader               = "Link"
+	maxPoolSize                 = 32 // The max number of parallel delete requests recommended by ACR server
+	mediaTypeDockerManifestList = "application/vnd.docker.distribution.manifest.list.v2+json"
+	mediaTypeArtifactManifest   = "application/vnd.oci.artifact.manifest.v1+json"
+	mediaTypeImageManifest      = v1.MediaTypeImageManifest
+	mediaTypeImageIndex         = v1.MediaTypeImageIndex
```

Comment:
`v1.MediaTypeImageManifest` and `v1.MediaTypeImageIndex` are already constants. We don't need to define them again.

Reply:
done